astropy/modeling/functional_models.py

```diff
@@ -837,6 +837,12 @@ class Lorentz1D(Fittable1DModel):
     x_0 = Parameter(default=0)
     fwhm = Parameter(default=1)
 
+    @property
+    def stddev(self):
+        """Standard deviation based on FWHM."""
+        from ..stats.funcs import gaussian_fwhm_to_sigma
+        return self.fwhm * gaussian_fwhm_to_sigma
+
```

Comment:
Indeed, you are correct! http://mathworld.wolfram.com/FullWidthatHalfMaximum.html . I do have a problem getting a bounding box that contains all the flux to agree with a bounding box that plots the central feature nicely. But to use this model to represent a spectrum, and enable flux integration, I do need some kind of optimal integration limit. I see that Scipy has `gamma` function, but not sure if that is really necessary for bounding box computation. Any ideas?